src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfAttributesWithValue = sourceString.split(';');
+  const trimmedArrayOfCouples
+    = arrayOfAttributesWithValue.map(line => line.trim().split(':'));
+  const filtredEmptyElements
+    = trimmedArrayOfCouples.filter(value => value.length !== 1);
+
+  const styles = filtredEmptyElements.reduce((prev, attribute) => {
```

Comment:
Можеш використати деструктуризацію
```suggestion
  const styles = filtredEmptyElements.reduce((prev, [key, value]) => {
```